src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/config.jelly

```diff
@@ -23,15 +23,15 @@
   </f:entry>
 
   <f:entry field="command" title="${%Command to run}">
-    <f:textbox default="/bin/sh -c"/>
+    <f:textbox default="sleep"/>
   </f:entry>
 
   <f:entry field="args" title="${%Arguments to pass to the command}">
-    <f:textbox default="cat"/>
+    <f:textbox default="9999999"/>
   </f:entry>
 
   <f:entry field="ttyEnabled" title="${%Allocate pseudo-TTY}">
-    <f:checkbox default="true"/>
+    <f:checkbox default="talse"/>
```

Comment:
typo: does `talse` evaluate to `false` anyway?

Reply:
Good catch: #957

> does `talse` evaluate to `false` anyway?

I think so, yes. For that matter, it is the default for the `default`.